src/types/num.rs

```diff
@@ -420,6 +422,49 @@ mod bigint_conversion {
             let zero: BigInt = FromPyObject::extract(fib.call1("fib", (0,)).unwrap()).unwrap();
             assert_eq!(zero, BigInt::from(0));
         }
+
+        /// `OverflowError` on converting python int to BigInt
+        #[test]
+        fn issue_629() {
```

Comment:
I don't think this name is very understandable.
How about `check_overflow` or so?

Reply:
sounds good, will change in next revision